strelets/adapter/swarm_prepare.go

```diff
@@ -62,7 +62,7 @@ func getSecretReference(containerName string, secretId string, secretName string
 func getContainerSpec(imageName string, secrets []*swarm.SecretReference, mounts []mount.Mount) *swarm.ContainerSpec {
 	command := []string{
 		"/opt/orbs/orbs-node",
-		"--silent",
+		//"--silent", TODO enable in tests only?
```

Comment:
This may overwhelm docker log storage. We don't need these logs to go in stdout.